src/Adapters/Redis/RedisConnectionInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Lamoda\Metric\Adapters\Redis;
+
+interface RedisConnectionInterface
+{
+    /**
+     * @return MetricDto[]
+     */
+    public function getAllMetrics(): array;
+
+    public function adjustMetric(string $key, float $value, string $tags): void;
+
+    public function setMetric(string $key, $value, string $tags): void;
```

Comment:
I hope we can drop this method since we have `setMetrics` with the same functionality

Reply:
Dropped